mlflow/environment_variables.py

```diff
@@ -841,6 +841,36 @@ def get(self):
     "MLFLOW_LOCK_MODEL_DEPENDENCIES", False
 )
 
+#: Specifies whether to enable delta archival for trace data to TraceServer.
+#: When set to True, traces will be exported to both MLflow V3 and TraceServer in parallel.
+#: (default: ``False``)
+MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL = _BooleanEnvironmentVariable("MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL", False)
+
+#: Specifies the name of the table to ingest spans into for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_SPANS_TABLE = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_SPANS_TABLE", str, None)
+
+#: Specifies the personal access token for authentication for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_TOKEN", str, None)
+
+#: Specifies the URL of the ingest API for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_INGESTION_URL = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_INGESTION_URL", str, None)
+
+#: Specifies the URL of the Databricks workspace for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_WORKSPACE_URL = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_WORKSPACE_URL", str, None)
+
+#: Specifies the name of the events table to ingest assessments into for TraceServer.
+#: This is only used when MLFLOW_TRACING_ENABLE_DELTA_ARCHIVAL is set to True.
+#: (default: ``None``)
+MLFLOW_TRACING_DELTA_ARCHIVAL_EVENTS_TABLE = _EnvironmentVariable("MLFLOW_TRACING_DELTA_ARCHIVAL_EVENTS_TABLE", str, None)
```

Comment:
As @B-Step62 points out, I think it would be simpler / clearer to add the ingest URL as an argument to the enablement API in https://github.com/mlflow/mlflow/pull/16654/files, rather than requiring users to set multiple env vars.

We can always add env var support in the future, but we may never need to, since trace ingestion to delta should automatically "just work" at some point.